index.html

```diff
@@ -2582,7 +2582,7 @@ <h3>
           <a>purpose</a>.
         </p>
         <p class="note">
-          For example, an icon with purpose "<a>monochrome</a>" could be used
+          For example, an icon with purpose "<a>single_color</a>" could be used
```

Comment:
Because we like to confuse everyone, I think we want a “-“ instead of “_” (see also orientation).